sdk/nodejs/yarn.lock

```diff
@@ -0,0 +1,3079 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Introducing lockfiles to the Node SDK is something that was deliberately avoided thus far AFAIK. Can this change be called out, separate PR perhaps? Which process will be put in place to update the lockfile, are we enabling dependabot?

Reply:
Yes, this commit should be pulled into a separate PR.

Reply:
Extracted to #10815